plugins/woocommerce-blocks/tests/e2e/tests/add-to-cart-form/add-to-cart-form.block_theme.spec.ts

```diff
@@ -57,13 +57,24 @@ test.describe( `${ blockData.name } Block`, () => {
 		admin,
 		editor,
 		editorUtils,
+		requestUtils,
 	} ) => {
-		// Add to Cart with Options in the Site Editor is only available as inner block of the Single Product Block except for the Single Product Template
+		// Add to Cart with Options in the Site Editor is only available as
+		// inner block of the Single Product Block except for the Single Product
+		// Template
+		const template = await requestUtils.createTemplate( 'wp_template', {
+			slug: 'product-catalog',
+			title: 'Custom Product Catalog',
+			content: 'placeholder',
+		} );
```

Comment:
Just to confirm, the goal of creating a new template in each test is to avoid side-effects between tests that modify the same template? Or is it for another reason?

Reply:
Ooh, makes sense. Thanks for clarifying!